scripts/tools/ExportLoginCommand.php

```diff
@@ -0,0 +1,213 @@
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2019 (original work) Open Assessment Technologies SA
+ *
+ */
+
+namespace oat\taoResultExports\scripts\tools;
+
+
+use core_kernel_classes_Resource;
+use oat\generis\model\OntologyAwareTrait;
+use oat\oatbox\extension\script\ScriptAction;
+use oat\oatbox\filesystem\FileSystemService;
+use oat\taoResultExports\model\export\ExportLoginService;
+use oat\taoResultExports\model\export\renderer\CsvRenderer;
+use oat\taoResultExports\model\export\renderer\RendererInterface;
+use oat\taoResultExports\model\export\renderer\StdOutRenderer;
+
+/**
+ * Class GenerateLoginCsvFile
+ * @package oat\taoResultExports\scripts\tools
+ *
+ * usage : sudo -u www-data php index.php 'oat\taoResultExports\scripts\tools\ExportLoginCommand' -d myDelivery
+ */
+class ExportLoginCommand extends ScriptAction
+{
+    use OntologyAwareTrait;
+
+    private $availableRenderers = [
+        CsvRenderer::SHORT_NAME,
+        StdOutRenderer::SHORT_NAME,
+    ];
+
+    protected function provideDescription()
+    {
+        return 'TAO Results - generate Csv of logins';
+    }
+
+    protected function provideOptions()
+    {
+        return [
+            'delivery' => [
+                'prefix'      => 'd',
+                'longPrefix'  => 'delivery',
+                'required'    => true,
+                'description' => 'List of deliveries to export (separated by comma)',
+            ],
+            'renderer' => [
+                'prefix'       => 'r',
+                'longPrefix'   => 'renderer',
+                'description'  => 'Renderer of the report [' . implode(',', $this->availableRenderers) . ']',
+                'defaultValue' => CsvRenderer::SHORT_NAME,
+                'cast'         => 'string',
+            ],
+            'withHeaders' => [
+                'prefix'       => 'wh',
+                'longPrefix'   => 'with-headers',
+                'flag'         => true,
+                'description'  => 'Export csv with headers',
+                'defaultValue' => false,
+                'cast'         => 'boolean',
+            ],
+            'withoutTimestamp' => [
+                'longPrefix'    => 'without-timestamp',
+                'prefix'        => 'wt',
+                'flag'          => true,
+                'description'   => 'Setting this flag would mean that export file won\'t have timestamp postfix in filename',
+                'defaultValue'  => false,
+                'cast'          => 'boolean',
+            ],
```

Comment:
Could you please add the `prefix` option so we can determine upfront the name of the file?

Reply:
it's added now